docs/Documentation/Drivers/Image/ForcingData.md

```diff
@@ -1,2 +1,103 @@
-!!! Warning
-    Docs not written yet.
+# VIC Forcing File
+
+The VIC Image Driver requires a NetCDF file with gridded subdaily forcings.  The required forcing variables and units are listed below (and must also be specified in the global parameter file):
+
+#### Meteorological Forcings, Required in all simulations:
+
+| Variable   | Description                         | Units           |   
+|------------|-------------------------------------|---------------- |
+| AIR_TEMP   | Average air temperature             | C               |   
+| PREC       | Total precipitation (rain and snow) | mm              |   
+| PRESSURE   | Atmospheric pressure                | kPa             |   
+| SWDOWN     | Incoming shortwave radiation        | W/m<sup>2</sup> |
+| LWDOWN     | Incoming longwave radiation         | W/m<sup>2</sup> |
+| VP         | Vapor pressure                      | kPa             |   
+| WIND       | Wind speed                          | m/s             |   
+
+Example output from `ncdump -h forcingfile.nc` should look like this: 
+
+```
+netcdf Stehekin_image_test.forcings_10days.1949 {
```

Comment:
`ncdump -h forcingfile.nc` --> `ncdump -h Stehekin_image_test.forcings_10days.1949`

which reminds me, you should mention that the files must be provided in chunks of individual years (e.g. `prefix.$year.nc`).